base/model_loader.py

```diff
@@ -0,0 +1,87 @@
+import json
+import os
+
+import mxnet as mx
+
+import tempfile
+import urllib
+import time
+
+
+def load_model(symbol_filename, params_filename):
+    """
+    Load model checkpoint from file.
+    :return: (symbol, arg_params, aux_params)
+    symbol : Symbol
+        The network configuration.
+    arg_params : dict of str to NDArray
+        Model parameter, dict of name to NDArray of net's weights.
+    aux_params : dict of str to NDArray
+        Model parameter, dict of name to NDArray of net's auxiliary states.
+    """
+    symbol = mx.symbol.load(symbol_filename)
+    save_dict = mx.nd.load(params_filename)
+    arg_params = {}
+    aux_params = {}
+    for key, value in save_dict.items():
+        tp, name = key.split(':', 1)
+        if tp == 'arg':
+            arg_params[name] = value
+        if tp == 'aux':
+            aux_params[name] = value
+    return symbol, arg_params, aux_params
+
+
+def download_url(url, target, retries=2, base_retry_interval=0.01):
```

Comment:
you need to take retries and base_retry_interval from a config file. Please do it as a separate PR.